app/javascript/components/miq-toolbar.jsx

```diff
@@ -45,6 +45,40 @@ const getParams = (urlParms, sendChecked) => {
   return _.filter(params).join('&') || undefined;
 };
 
+const convertMultParamsToArray = (fullParamString) => {
```

Comment:
Paired with @MelsHyrule on extracting this to a utility file, then writing specs for it...hopefully she'll have a commit for that soonish.  

---

Also, I had misunderstood what this method was for...I thought it parsed the string and returned an Array, for which I thought there would be some sort of public package that could do this.  However, instead it takes a param string with multi values and adjusts it to another param string but with the `[]` format as expected by Rails.  So, I think we should rename the method accordingly, but alas Naming Is Hard™ 

- `convertMultParamsToArrayStyle`
- `convertMultParamsToRailsMultParams`
- `convertMultParamsToRailsMultParamsArrayStyle`

🤷 

Aside, why `MultParams` as opposed to `MultiParams`?

Reply:
Pushed up recently the spec test draft 1, and I like `convertMultParamsToRailsMultParams` as a name so will push up a renaming commit later if no one has strong opinions against this being the name. 


> Aside, why MultParams as opposed to MultiParams?

Because Naming Is Hard™